src/main/java/de/qabel/desktop/ui/accounting/identitycontextmenu/IdentityContextMenuController.java

```diff
@@ -50,6 +66,16 @@
     @Override
     public void initialize(URL location, ResourceBundle resources) {
         resourceBundle = resources;
+        createButtonsGraphics();
+    }
+
+    private void createButtonsGraphics() {
+        editButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/pencil.png"))));
+        removeButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/delete.png"))));
+        exportButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/upload.png"))));
+        privateKeyButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/qrcode.png"))));
+        publicKeyQRButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/qrcode.png"))));
+        publicKeyEmailButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/email.png"))));
```

Comment:
Please extract this repitition (like the `optionImage` method in the RemoteFSController) and make the graphics static. With this implementation, they get loaded everytime someone opens an IdentityContextMenu. Something that is probably to happen multiple times.